src/main/resources/mamute-messages.properties

```diff
@@ -661,13 +661,13 @@ button_hint.codes.text = Code
-button_hint.html.hint = <p>If you need something that the markdown can't do, use HTML. Note that we support only a frw HTML tags!</p><pre>Pegar sal da m�o de outros da &lt;strike&gt;azar&lt;/strike&gt;.</pre><p>O Markdown � esperto o bastante para n�o estragar seu HTML:</p><pre>&lt;b&gt;Markdown funciona *fine* aqui.&lt;/b&gt;</pre><p>Tags de bloco do HTML possuem algumas restri��es:</p><ol><li>Elas precisam ser separadas do texto em volta com linhas em branco.</li><li>� necess�rio que o inicio e o fim das tags n�o estejam identados.</li><li>Markdown n�o pode ser usado em blocos de HTML.</li></ol><pre>&lt;pre&gt;<br>	Voc� &lt;em&gt;n�o pode&lt;/em&gt; use Markdown here.<br>&lt;/pre&gt;</pre>
+button_hint.lists.hint = <p>Both ordered lists and non-ordered lists are possible:</p><div class="button-hint-col"><pre>-	Use dashs for bullets<br>+	Or a plus sign<br>*	Or an asterisk<br><br>1.	Ordered lists are easy<br>2.	Markdown will enumerate them for you<br>7.	Then item 3 will be 3.</pre></div><div class="button-hint-col"><pre>1.	Lists inside an item:<br>-	Indent with 4 spaces.<br>*	Indent with 8 spaces.<br>-	Four spaces again.<br><br>2.	You can have multiple<br>paragraphs in an item.<br>Only verify the indentation.</pre></div>
+button_hint.blockquotes.hint = <div class="button-hint-col"><pre>&gt; Make quotes by inserting<br>&gt; \u201C&gt;\u201D at the begin of each line.<br>&gt;<br>&gt; Other formats also work here:<br>&gt;<br>&gt; 1. Lists or<br>&gt; 2. Titles:<br>&gt;<br>&gt; ## Quotes title ##<br></pre></div><div class="button-hint-col"><p>You can also add quotes for quotes:</p><pre>&gt; The super quote is indented<br>&gt; &gt; The child quote is more indented<br>&gt; &gt; &gt; &gt; You can add as many quotes as you want.<br></pre></div>
+button_hint.codes.hint = <p>To create code blocks, add three backticks (```) before and after the code:</p><pre>	The code will be shown in a monospaced font.<br><br>	Markdown and HTML will be disabled in blocks of code:<br>	&lt;i&gt;This is not italic&lt;/i&gt;, and [this is not a link]({domain})</pre><p>To create only a line or word of code, use only 1(one) backtick:</p><pre>The character `$` is only a shortcut to `window.jQuery`.</pre>
+button_hint.html.hint = <p>If you need something that the markdown can't do, use HTML. Note that we support only a few HTML tags!</p><pre>Pegar sal da m�o de outros da &lt;strike&gt;azar&lt;/strike&gt;.</pre><p>O Markdown � esperto o bastante para n�o estragar seu HTML:</p><pre>&lt;b&gt;Markdown funciona *fine* aqui.&lt;/b&gt;</pre><p>Tags de bloco do HTML possuem algumas restri��es:</p><ol><li>Elas precisam ser separadas do texto em volta com linhas em branco.</li><li>� necess�rio que o inicio e o fim das tags n�o estejam indentados.</li><li>Markdown n�o pode ser usado em blocos de HTML.</li></ol><pre>&lt;pre&gt;<br>	Voc� &lt;em&gt;n�o pode&lt;/em&gt; use Markdown here.<br>&lt;/pre&gt;</pre>
```

Comment:
Eek. Are you sure you use UTF-8 encoding for these files? Normally properties files are latin1 encoded when using java.util.Properties to load them. See http://docs.oracle.com/javase/7/docs/api/java/util/Properties.html#load(java.io.InputStream)